### ISSUE TITLE
No telemetry Data

### ISSUE THREAD

**Issue by dentex** [yaapu/HorusMappingWidget#2]
Ciao Alessadro, 
I was trying to setup your widget; my setup:
_Jumper T16 radio
R9M module
R9MM receiver
**FPort**;
Betaflight_

The telemetry works, I have all the data on the radio:
![photo_2020-03-31_11-46-41](https://user-images.githubusercontent.com/2931982/78013870-66099e00-7347-11ea-8f92-6f79b6b00100.jpg)

The widgets are not showing anything. 
![photo_2020-03-31_11-46-39](https://user-images.githubusercontent.com/2931982/78013895-6f930600-7347-11ea-931c-f98b5336d28d.jpg)

![photo_2020-03-31_11-46-38](https://user-images.githubusercontent.com/2931982/78012533-6143ea80-7345-11ea-8748-629d81443639.jpg)

As a "secondary" problem I also do not have maps, but I'm sure I loaded the correct tiles.
![photo_2020-03-31_11-46-35](https://user-images.githubusercontent.com/2931982/78013919-77eb4100-7347-11ea-8200-2f63b187afd5.jpg)

I checked for my location and its tiles manually and they are present on the sdcard.

I'm sure it's something on my side.
Do you have any suggestion?
Please let me know if you need more info.

Grazie.

**Comment by yaapu** [comment]
Hi the Full Telemetry Widget does not work with Betaflight, it requires ardupilot.
The Mapping Widget does not find the tiles so you probably did something wrong with the tiles.
The tiles engine is used by several on the full widget so the most probable cause is an error in your tiles db, please follow the instructions step by step:
- download (the correct zoom levels, from your screenshot I'd say -2,-1 and0)
- resize
- copy

**Comment by dentex** [comment]
OK, so I have to use the Mapping widget only. Great.
For the map tiles... I'll check one more time. 

Thanks.   :)

**Comment by dentex** [comment]
I found my way through it!
a)The first time I copied the file on the sdcard, probably something got corrupted, or I don't know...; so I though I needed also the FrSkyTelemetry script (that instead supports Ardupilot only)... 
b)And for the tiles, that's another story. I downloaded/edited them in Linux, under wine... it turned out the script didn't like it. 

So:
fix-a) I just deleted everything and copied again the script files (Mapping widget only) and;
fix-b) I did again the tiles work in VirtualBox (Win7 machine).

That finally worked:

![photo_2020-03-31_21-16-09](https://user-images.githubusercontent.com/2931982/78066592-8a8c6700-7395-11ea-87a0-ac68a5dc1ff1.jpg)

Thanks Alessadro for the wonderful job.
